src/components/Redoc/Redoc.tsx

```diff
@@ -22,10 +22,23 @@ import {
 import { SearchBox } from '../SearchBox/SearchBox';
 import { StoreProvider } from '../StoreBuilder';
 
+import { VersionSelector } from '../VersionSelector';
+
 export interface RedocProps {
   store: AppStore;
 }
 
+/** WILL REMOVE BEFORE MERGING!!! */
+/** Mock data that will be passed to store from cli */
+export const versionData = {
```

Comment:
We could also move this into a `mockVersionData.json` file somewhere if it might be useful for testing purposes (unit and manual, until the CLI is set up)